codeforamerica/vita-min
# GYR1-428 clear code on save

## [GYR1-428](https://codeforamerica.atlassian.net/browse/GYR1-428)
## What was done?
 * Clear form code on save
## How to test?
Test on heroku. The input box in the Unique Links section on the orgs page should clear when save is clicked

<img width="863" alt="image" src="https://github.com/codeforamerica/vita-min/assets/17094895/0fe16269-9f5c-4c2f-8440-7d2dc783955a">

<img width="708" alt="image" src="https://github.com/codeforamerica/vita-min/assets/17094895/bdfc8d0e-c102-4171-903a-ab955662af0b">



[GYR1-428]: https://codeforamerica.atlassian.net/browse/GYR1-428?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
